.github/workflows/build.yml

```
@@ -1,398 +1,275 @@
+          for script in authaws ssm src/*.sh tools/*.sh scripts/*.sh 01_install.sh 02_uninstall.sh; do
+            if [ -f "$script" ]; then
+              bash -n "$script" || exit 1
+            fi
+          done
+          echo "Shell script syntax is valid"
+
+      - name: Notify on shell test failure
+        if: failure() && matrix.os == 'ubuntu-latest' && github.event_name == 'pull_request'
+        env:
+          GOOGLE_CHAT_WEBHOOK: ${{ secrets.GOOGLE_CHAT_WEBHOOK }}
+        run: |
```

actionlint (1.7.9)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)